syndesis/plugins/org.fusesource.ide.syndesis.extension.ui/src/org/fusesource/ide/syndesis/extensions/ui/l10n/messages.properties

```diff
@@ -63,6 +63,7 @@ newProjectWizardExtensionDetailsPageOptionalTagsFieldHint = optional comma separ
 newProjectWizardExtensionDetailsPageErrorMissingExtensionId = Please specify a unique extension id.
 newProjectWizardExtensionDetailsPageErrorInvalidExtensionId = The extension id must not contain spaces.
 newProjectWizardExtensionDetailsPageErrorMissingExtensionVersion = Please specify a version for your extension.
+newProjectWizardExtensionDetailsPageErrorInvalidExtensionVersion = The provided extension version is not a valid version.
```

Comment:
it would be nice to precise what is the expected pattern

Reply:
Agreed